rollup.config.mjs

```diff
@@ -0,0 +1,76 @@
+/**
```

Comment:
This file should replace `rollup.config.js` correct? If so, that file needs to be deleted here.

Reply:
:facepalm: forgot to add the deleted file to the commit, fixed in 31ce003